### PR TITLE
#6209 added aliases for javascript and html magic command

### PR DESCRIPTION
**Pull request by lmitusinski**
Default matchCommand logic has been separated to MagicCommandFunctionality interface as default methods.

 If magic command require to use multiple aliases it can override getMagicCommandAliases() or if it require custom command matching it can override matchCommand()

### CODE REVIEW

**Comment by scottdraves** [comment]
This is good I like how `matchCommand` is factored out.
However, `%lsmagic` needs to report the new aliases.

**Comment by scottdraves** [comment]
another possible approach: don't create the notion of "aliases" in the code at all.
just define two magics "js" and "javascript" that both happen to call the same implementation?

**Comment by lmitusinski** [comment]
According to your comment i have created aliases as new magic commands by extending original magic commands and overwriting their names.

File: kernel/base/src/test/java/com/twosigma/beakerx/KernelTest.java [hunk 1]
```diff
@@ -36,23 +36,7 @@
...
-import com.twosigma.beakerx.kernel.magic.command.functionality.HtmlMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.JavaScriptMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.LoadMagicMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.LsMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.TimeCellModeMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.TimeItCellModeMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.TimeItLineModeMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.TimeLineModeMagicCommand;
-import com.twosigma.beakerx.kernel.magic.command.functionality.UnImportMagicCommand;
+import com.twosigma.beakerx.kernel.magic.command.functionality.*;
```
Comment (scottdraves) [inline kernel/base/src/test/java/com/twosigma/beakerx/KernelTest.java]: wildcard is not allowed by our java style guide.

Comment (scottdraves) [reply kernel/base/src/test/java/com/twosigma/beakerx/KernelTest.java]: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

**Pull request merged**